lib/config/presets/internal/workarounds.ts

```diff
@@ -79,7 +79,7 @@ export const presets: Record<string, Preset> = {
     description: 'Limit Java runtime versions to LTS releases',
     packageRules: [
       {
-        allowedVersions: '/^(?:8|11|17|21|25|29)(?:\\.|$)/',
+        allowedVersions: '/^(?:8|11|17)',
```

Comment:
```suggestion
        allowedVersions: '/^(?:8|11|17)(?:\\.|-|$)/',
```
- it's too wide otherwise
- missing regex end marker